### DISCUSSION TITLE
GoPro for construction time-lapse, need solution for time problem.

### DISCUSSION THREAD

**Discussion by long2391** [gopro/labs#1120] [Q&A]
I am using GoPro 9 BLACK and newest labs firmware for long-term construction time-lapse. I got some problems: 
1. **USB Trigger** (I solved it by make a command) 
2. **Time** (After 8 hours without power, GoPro must be setup time manually or via QuikApp, how i make this auto update time) 
3. **I don't know that my GoPro is working or not** because it placed in hard-to-reach position? How i can make it upload files to GG Drive or any signal? 

Tysm for helping!

**Comment by dnewman-gpsw** [comment]
H9 is not a great choice for long form timelapse less you just provide continuous power, using the default timelapse features.  The issue is the wake time event is not reliable not beyond an average of 300-400 starts. H12 and H13 addresses that issue.
